### DISCUSSION TITLE
How to open terminal in bottom across multiple windows?

### DISCUSSION THREAD

**Discussion by AstrickHarren** [NvChad/NvChad#2860] [Q&A] [answered]
In version 2.5, the position option to use is either `sp` or `vsp`. 

Version 2.0 supports positions like bottom, which opens the terminal at the bottom but across all split windows.

**Comment by siduck** [comment]
@AstrickHarren  u sure nvterm did that? i just tried `:lua require("nvterm.terminal").new "horizontal"` and it opened split in below the window, not across all

**Reply by AstrickHarren** [reply, marked as answer]
Yeah, I'm pretty sure. I think you can go with position as `botright`.

**Reply by siduck** [reply]
@AstrickHarren  thanks for it! i'll include a new option for it :)) , just wait a day or two. will let you know
